### ISSUE TITLE
Displays nothing

### ISSUE THREAD

**Issue by dpo** [elmasse/hyper-markdown-preview#4]
Hyper 2.0.3
OSX 10.12.6

I'm also using hyper-apex 0.1.1 but disabling it did not resolve the issue. I don't use any other plugin. Config file: https://gist.github.com/a9217a1ddfcf9e3443f3bbba0622aa35.

Thanks.

**Comment by elmasse** [comment]
There is an issue with electron not displaying the content when using createObjectURL (which this extension relies on) https://github.com/electron/electron/issues/10438

There is a really ugly workaround that might not work properly in certain scenarios. I have to test it with hyper 2.x yet, but I'd love to get that electron issue fixed.

I'm gonna test on hyper 2.x and see if that is related to confirm.

**Comment by elmasse** [comment]
I added back the createObjectURL and tested it on hyper 1.x and it seems to be working fine. Could you try v1.1.3 and see if this is solved?

**Comment by dpo** [comment]
Thanks for your efforts! I cloned this repository under `~/.hyper_plugins/local` and listed it in the `localPlugins` array at the end of `/.hyper.js`. Then I went into `~/.hyper_plugins/local/hyper-markdown-preview` and ran `apm install`. According to the dev tools, version 1.1.3 loaded. However, I still can't view a Markdown file. Did I install things correctly?

**Comment by elmasse** [comment]
Yup, that's correct. I had a false positive I think. Not sure why it worked. I refreshed hyper and now is back to the blank screen. I'm trying to install the Hyper 2.x version but running yarn just never finishes :/
